src/plugins/query_enhancements/common/utils.ts

```diff
@@ -49,7 +49,11 @@ export const handleFacetError = (response: any) => {
 
 export const fetch = (context: EnhancedFetchContext, query: Query, aggConfig?: QueryAggConfig) => {
   const { http, path, signal } = context;
-  const body = JSON.stringify({ query: { ...query, format: 'jdbc' }, aggConfig });
+  const body = JSON.stringify({
```

Comment:
Should this use `stringify` from `@osd/std`?

Reply:
I am going to go with your suggestion here since I am new to this part of the code. @ashwin-pc let me know if think this is wrong